### PR TITLE
MR-09-17: Delete Trainer

### PR DESCRIPTION
**Pull request by OscarLucero**
Added delete trainer.

### CODE REVIEW

File: src/routes/trainer.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const router = express.Router();
 router
   .put('/:id', validateUpdate, trainerController.updateTrainer)
   .get('/', trainerController.getAllTrainers)
+  .delete('/:id', trainerController.deleteTrainer)
```
Comment (danielninfi) [inline src/routes/trainer.js:10]: this is ok!

Comment (danielninfi) [inline src/routes/trainer.js:10]: this is ok for me

File: src/controllers/trainer.js [hunk 2]
```diff
@@ -73,4 +73,27 @@ const updateTrainer = (req, res) => {
...
+      });
+    })
+    .catch((error) => res.status(400).json({
+      message: 'There was an mistake!',
+      error,
+    }));
+};
+
+module.exports = {
+  getAllTrainers, getTrainerById, updateTrainer, deleteTrainer,
```
Comment (danielninfi) [inline src/controllers/trainer.js:98]: this generated a problem for me (it does not mean that it generated it for you).
to solve it the recommended identation is 
module.exports = {
 getAllTrainers, 
getTrainerById, 
updateTrainer, 
deleteTrainer,
};

**Review by danielninfi — APPROVED** [review]

File: src/controllers/trainer.js [hunk 3]
```diff
@@ -73,4 +73,27 @@ const updateTrainer = (req, res) => {
...
+        return res.status(404).json({
+          message: `Trainer with the id: ${id} was not found, please try with another one`,
+          error: true,
+        });
+      }
+      return res.status(200).json({
+        message: `Trainer with the id: ${id} was successfully deleted.`,
+      });
+    })
+    .catch((error) => res.status(400).json({
```
Comment (NicoCagnina) [inline src/controllers/trainer.js:91]: this should be 500

**Review by NicoCagnina — CHANGES_REQUESTED** [review]

File: src/controllers/trainer.js (hunk 3 above)
Comment (OscarLucero) [reply src/controllers/trainer.js]: Solved!

**Review by NicoCagnina — APPROVED** [review]

**Review by JuanMornik — APPROVED** [review]

**Pull request merged**